.github/workflows/pipeline-perf-test-continuous.yml

```
@@ -100,8 +100,14 @@ jobs:
         - name: Run idle state performance tests
           run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2086:info:5:50: Double quote to prevent globbing and word splitting

(shellcheck)